doc/doc.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -eu
+
+DOC_VERSION=4
+DOC_PATH=/sdk/php/4
+
+# Used by vuepress
+export DOC_DIR=$DOC_VERSION
+export SITE_BASE=$DOC_PATH/
+
+# Used to specify --no-cache for example
+ARGS=${2:-""}
+
+if [ ! -d "./$DOC_DIR" ]
```

Comment:
I find this impractical: scripts should work whatever our running directory is. 

Instead, what do you think of the following:

```bash
export CWD=`dirname "$0"`
```

And then use `$CWD/<path>` instead of `./<path>` in the remaining of this script?